src/cpp/cpp_declarator_converter.cpp

```diff
@@ -258,7 +258,7 @@ void cpp_declarator_convertert::combine_types(
         if(decl_parameter.type()!=symbol_parameter.type())
         {
           // The 'this' parameter of virtual functions mismatches
-          if(i!=0 || !symbol_code_type.get_bool("#is_virtual"))
+          if(i != 0 || !symbol_code_type.get_bool(ID_C_is_virtual))
```

Comment:
Should the C++ front-end be generating ID_C_* things for C++ features?

Reply:
That `_C_` isn't C the language, it's "Comment". C the language seems to usually use a lowercase 'c' in irep identifiers.